movie_REST_API/movie_REST_API/Controllers/InfoMovieViewController.swift

```diff
@@ -0,0 +1,278 @@
+        URLSession.shared.dataTask(with: url) { data, _, error in
+            
+            if let error = error {
+                print(Constant.errorDataTaskString, error.localizedDescription)
+                return
+            }
+            
+            guard let data = data else {
+                print(Constant.emptyDataString)
+                return
+            }
+            
+            DispatchQueue.main.async {
+                if let image = UIImage(data: data) {
+                    self.movieImageView.image = image
+                }
+            }
+        }.resume()
+    }
+    
+    private func createUI() {
+        view.backgroundColor = .black
+        
+        let backgroundImage = UIImageView(frame: UIScreen.main.bounds)
+        backgroundImage.image = UIImage(named: Constant.baseImageName)
+        backgroundImage.contentMode = .scaleAspectFill
+        view.insertSubview(backgroundImage, at: 0)
+
+        addUI()
+    }
+    
+    private func addUI() {
+        view.addSubview(nameFilmLabel)
+        view.addSubview(movieImageView)
+        view.addSubview(descpriptionTextView)
+        view.addSubview(goToWebButton)
+    }
+    
+    private func createNavController() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: Constant.starImageName),
+            style: .done,
+            target: self,
+            action: #selector(starAction)
+        )
+        navigationItem.rightBarButtonItem?.tintColor = .purple
+        navigationController?.navigationBar.tintColor = UIColor.black
+
+    }
+    
+    @objc private func starAction() {
+        if isPress == false {
+            tapOkButton(title: Constant.addFavouriteString, message: Constant.emptyString, handler: nil)
+            navigationItem.rightBarButtonItem?.image = UIImage(systemName: Constant.starFillImageName)
+            isPress = true
+        } else {
+            tapOkButton(title: Constant.deleteFavouriteString, message: Constant.emptyString, handler: nil)
+            navigationItem.rightBarButtonItem?.image = UIImage(systemName: Constant.starImageName)
+            isPress = false
+        }
+    }
+
+    private func createCollectionView() {
+        imageCollectionView.register(InfoMovieCell.self, forCellWithReuseIdentifier: Constant.cellIdentifier)
+        imageCollectionView.delegate = self
+        imageCollectionView.dataSource = self
+        view.addSubview(imageCollectionView)
+    }
+
+    private func setImageConstraints() {
+        NSLayoutConstraint.activate([
+            movieImageView.topAnchor.constraint(equalTo: nameFilmLabel.bottomAnchor, constant: 10),
+            movieImageView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
+            movieImageView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10),
+            movieImageView.heightAnchor.constraint(equalToConstant: 230)
+        ])
+    }
```

Comment:
```suggestion
    }
    
```
Ну как же что б Наташа отступы ставила после методов)